### PR TITLE
#361 loan helpers test fails intermittently

### PR DESCRIPTION
**Pull request by kevinluo201**
# Related issues
[https://github.com/rubyforgood/circulate/issues/361](https://github.com/rubyforgood/circulate/issues/361)

# Summary
Time sensitive test is better to manually set the time to prevent timezone problem.

# Screenshots
before
image
after
image

### CODE REVIEW

**Comment by rkim00** [comment]
I thought it might have to do with timezones! Thanks!

File: test/helpers/loans_helper_test.rb [hunk 1]
```diff
@@ -8,9 +8,11 @@ class LoansHelperTest < ActionView::TestCase
   end
 
   test "humanize_due_date returns tomorrow when due date is tomorrow" do
-    @loan = build(:loan, due_at: 1.days.from_now)
-
-    assert_equal "tomorrow", humanize_due_date(@loan)
+    Timecop.freeze(Date.new(2020, 1, 1)) do
```
Comment (jim) [inline test/helpers/loans_helper_test.rb:11]: Can we use the built-in `travel_to` method from Rails instead of timecop here?

**Review by jim — CHANGES_REQUESTED** [review]
Thanks for taking a look at this and identifying a solution.

If possible, I'd like to avoid adding a gem dependency if we can accomplish the same thing using Rails built-in functionality. [Here's the doc page for travel_to](https://api.rubyonrails.org/classes/ActiveSupport/Testing/TimeHelpers.html#method-i-travel_to).

File: test/helpers/loans_helper_test.rb (hunk 1 above)
Comment (kevinluo201) [reply test/helpers/loans_helper_test.rb]: I removed the timecop.
I also found that the root cause is the `from_now` method which use Time.current as input.
Time.current return UTC time that's why it has timezone problem
https://apidock.com/rails/ActiveSupport/Duration/from_now.

I changed to use Date.today + 1.day, which always uses local time, so we don't need change Time.now anymore.

**Review by jim — APPROVED** [review]

**Comment by jim** [comment]
Thanks for fixing this for us!

**Pull request merged**